src/Tests/Microsoft.NET.Sdk.Razor.Tests/BuildIntegrationTest.cs

```diff
@@ -74,15 +74,18 @@ public void Build_ErrorInGeneratedCode_ReportsMSBuildError()
             var projectDirectory = CreateAspNetSdkTestAsset(testAsset);
 
             var filePath = Path.Combine(projectDirectory.Path, "Views", "Home", "Index.cshtml");
```

Comment:
I wonder if we should just make this a relative path and check for the existence of that instead of having the conditional check.

Reply:
I was hoping we would undo this PR after we get o# to fix their bug. But using a relative path seems like a good long term fix